common/test/acceptance/pages/lms/teams.py

```diff
@@ -324,6 +324,64 @@ def cancel_team(self):
         self.q(css='.create-team .action-cancel').first.click()
         self.wait_for_ajax()
 
+    def click_membership_button(self):
+        """Clicks the 'edit membership' button"""
+        self.q(css='.action-edit-members').first.click()
+        self.wait_for_ajax()
+
+    @property
+    def membership_button_present(self):
+        return self.q(css='.action-edit-members').present
+
+
+class EditMembershipPage(CoursePage):
+    """
+    Staff or discussion-privileged user page to remove troublesome or inactive
+    students from a team
+    """
+    def __init__(self, browser, course_id, team):
+        """
+        Set up `self.url_path` on instantiation, since it dynamically
+        reflects the current team.
+        """
+        super(EditMembershipPage, self).__init__(browser, course_id)
+        self.team = team
+        self.url_path = "teams/#teams/{topic_id}/{team_id}".format(
+            topic_id=self.team['topic_id'], team_id=self.team['id']
+        )
+
+    def is_browser_on_page(self):
+        """Check if we're on the team membership page for a particular team."""
+        self.wait_for_ajax()
+        if not self.url.endswith(self.url_path):
+            return False
+        if self.q(css='.edit-members').present:
+            return True
+        empty_query = self.q(css='.teams-main>.page-content>p').first
```

Comment:
If there are members on the team, we'll short-circuit out of this method by returning `True` on line 359. We need the additional explicit check on 362/363 because the `.edit-members` ul isn't present if there are no members.

The choice of `<ul class="edit-members">' versus '<p>' is made here: https://github.com/edx/edx-platform/blob/christina/remove-team-members/lms/djangoapps/teams/static/teams/js/views/edit_team_members.js#L36